spec/models/page_spec.rb

```diff
@@ -49,15 +49,12 @@
   end
 
   context "when callbacks are triggered after save" do
-    let(:page) { build(:page) }
-
-    before do
-      allow(Pages::BustCacheWorker).to receive(:perform_async)
-    end
+    let(:page) { create(:page) }
```

Comment:
shouldn't this be a `build(:page)` ? the reason why this test passes is because create also runs save

Reply:
I don't think it matters bc that block below asserts that a new job is enqueued on save. Whether it's a new record or already existing one is inconsequential